README.md

```diff
@@ -13,61 +13,118 @@
-each time (with no cross-cutting concerns) from PromLab's
-[PromQL Compliance Test Suite](https://promlabs.com/promql-compliance-tests/).
-
-For OpenTelemetry, Jaeger and Zipkin users, Promscale provides the ability to run deep analysis
-on traces using SQL while visualizing the distributed traces in Jaeger or Grafana.
-Install our lightweight [opentelemetry-demo](https://github.com/timescale/opentelemetry-demo)
-on your computer to quickly get a feel of what's possible. More details in
-[this blog post](https://www.timescale.com/blog/learn-opentelemetry-tracing-with-this-lightweight-microservices-demo/).
+<img alt="Promscale" src="docs/assets/promscale-logo.png" width="600px">
+
+Promscale is a unified metric and trace observability backend for Prometheus,
+Jaeger and OpenTelemetry built on PostgreSQL and TimescaleDB.
+
+Promscale serves as a robust and 100% PromQL-compliant Prometheus remote storage
+and as a durable and scalable Jaeger storage backend.
+
+Unlike other observability backends, it has a simple
+and easy-to-manage architecture with just two components: the Promscale
+Connector and the Promscale Database.
+
+[Learn more](https://docs.timescale.com/promscale/latest/about-promscale/#promscale-schema-for-metric-data) 
+about Promscale's architecture and how it works.
 
 <img src="docs/assets/promscale-arch.png" alt="Promscale Architecture Diagram" width="800"/>
 
-This repository contains the source code of the Promscale Connector. Promscale also requires that the Promscale Extension
-which lives in [this repository](https://github.com/timescale/promscale_extension) is installed in the TimescaleDB/PostgreSQL
-database. The extension sets up and manages the database schemas and provides performance and SQL query experience improvements.
+## Promscale for Prometheus
+
+Promscale solves two main Prometheus use cases:
+
+* **Efficient long-term trend analysis and capacity planning**<br/>
+Use Promscale as a durable long-term storage for Prometheus metrics. Reduce costs and
+improve query performance by leveraging [metric downsampling](https://docs.timescale.com/promscale/latest/downsample-data/)
+and [per-metric retention](https://docs.timescale.com/promscale/latest/manage-data/retention/#configure-data-retention-for-metrics) to 
+only keep the data you need for as long as you need it. 
+
+* **Single-pane-of-glass across all your Kubernetes clusters**<br/>
+Use Promscale as a centralized storage for all your Prometheus instances
+so you can easily query data across all of them in Grafana and centralize
+[alert management](https://docs.timescale.com/promscale/latest/alert/) and
+[recording rules](https://docs.timescale.com/promscale/latest/downsample-data/recording/).
+
+**Key features**: 100% PromQL-compliant, high availability, multi-tenancy, PromQL alerting and recording rules, per-metric retention.
+
+If you use Prometheus and are also familiar with PostgreSQL, then Promscale is a great choice for
```

Comment:
True and I think we need to talk to a specific target market instead of the entire Prometheus market. Casting a wide net is not working well for us anyway. If you are not familiar with Postgres, then yeah, you don't get much benefits from using our product. Most users that use us already pick us because of Postgres, why not make it even clearer?

This is like the timescaleDB positioning of Postgres for Timeseries. We are also narrowing who we are targeting and we do it on purpose.